etc/profile-m-z/strawberry.profile

```diff
@@ -0,0 +1,33 @@
+# Firejail profile for strawberry
+# Description: A music player and music collection organizer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include strawberry.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/strawberry
+noblacklist ${HOME}/.config/strawberry
+noblacklist ${MUSIC}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+include whitelist-var-common.inc
+
+caps.drop all
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+# blacklisting of ioprio_set system calls breaks strawberry
+seccomp !ioprio_set
+
+private-dev
```

Comment:
`disable-mnt`??, `private-bin strawberry,...`, `private-cache`, `private-etc …`,

Reply:
private-bin breaks strawberry when it needs to scan library, so I'm gonna need some more time finding what causes this,
also it needs ca-certificates, ssl, etc. to be able to communicate with services, lyrics sites', etc.

Reply:
Again `firejail --build` can be very helpful.

Reply:
basic networking: `private-etc ca-certificates,ssl,pki,crypto-policies,nsswitch.conf,resolv.conf,hosts,host.conf,hostname`

Reply:
Of course `strawberry-tagreader` was needed and also `/etc/fonts`, so it became this:
```
private-bin strawberry,strawberry-tagreader
private-etc ca-certificates,ssl,pki,crypto-policies,nsswitch.conf,resolv.conf,hosts,host.conf,hostname,fonts
```

Reply:
sorted:
```
private-etc ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,nsswitch.conf,pki,resolv.conf,ssl
```

Reply:
Sorted.